### ISSUE TITLE
Requiring at least one field doesn't work with 'model/field' naming

### ISSUE THREAD

**Issue by tsucaet** [formvalidation/support#94]
If using model[fieldname] as the input field name, the 'required at least one field' check doesn't work.

<pre>
&lt;input type="text" class="form-control emailAddress" name="user[primary_email]" />
&lt;input type="text" class="form-control emailAddress" name="user[secundary_email]" />
</pre>
referring: http://formvalidation.io/examples/requiring-at-least-one-field/

**Comment by nghuuphuoc** [comment]
Exactly! It's not possible to use name attribute with [selector](http://formvalidation.io/settings/#field-selector) option together at the moment.
The ```selector``` option is born to support nameless inputs.

I will update the example to help user who want to use the name attribute in that case.

**Comment by sujiangyin** [comment]
hello,I have seen the href :http://formvalidation.io/examples/requiring-at-least-one-field/.
But it only instructs the case that fileds have the same validation rules, what should I do for different rules in fileds,and required at least one field.Such as the phone number and email?

**Comment by sujiangyin** [comment]
my two field as follow:

'f-email': {
                 selector: '.f-email',
                 validators: {
                     notEmpty: {
                         message:{
                             en_US: 'You must enter at email address or phone number ',
                             zh_CN: '你必须填写邮箱或电话号码'
                         }
                     },
                     emailAddress: {
                         message: {
                             en_US: 'The input is not a valid email address',
                             zh_CN: '输入不是有效的邮箱地址'
                         }
                     },
                     remote: {
                    	 type: 'POST',      
                    	 url: "/cooka-user-web/isExistAccount?account=post",                                           
                         message: {
                             en_US: 'The email is already exist',
                             zh_CN: '邮箱已注册'
                         },
                         delay: 1000
                     }
                 }
             },
             'f-phone': {
                 selector: '.f-phone',
                 validators: {
                	  notEmpty: {
                          message:{
                              en_US: 'You must enter at email address or phone number ',
                              zh_CN: '你必须填写邮箱或电话号码'
                          }
                      },
                      intPhoneNumber: {
                          utilsScript: '/js/libs/FV/intl_tel/utils.js',
                          autoPlaceholder: true,
                          preferredCountries: 'cn,us,gb',
                          message: 'The phone number is not valid'
                      },
                      remote: {
                     	 type: 'POST',      
                     	 url: "/cooka-user-web/isExistAccount?account=post",                                           
                          message: {
                              en_US: 'The phone is already exist',
                              zh_CN: '电话已注存在'
                          },  
                          delay: 1000
                      }
                 }
             },

**Comment by huanggm** [comment]
@sujiangyin Hi，how do you solve this problem? I just encounter the same problem.
